Resources/Prototypes/Damage/modifier_sets.yml

```diff
@@ -22,7 +22,7 @@
     Slash: 10
     Piercing: 10
     Heat: 10
-    Structural: 10
+    Structural: 30
```

Comment:
```suggestion
    Structural: 80
```
![image](https://github.com/user-attachments/assets/05f7734c-facb-4c8b-87e5-457b3c24acc3)
Pickaxe actually does 52.5 damage on a wide swing, and if you stack modifiers you can get it up to 78.75 structural. So to make it pickaxe-proof, it needs to be MUCH higher.